src/Module/Conversation/Timeline.php

```diff
@@ -346,6 +346,10 @@ private function getRawChannelItems()
 
 		$items = [];
 		$result = $this->database->select('post-engagement', ['uri-id', 'created', 'owner-id', 'comments', 'activities'], $condition, $params);
+		if ($this->database->errorNo()) {
+			throw new \Exception($this->database->errorMessage(), $this->database->errorNo());
```

Comment:
Shouldn't we do this from the database library itself?

Reply:
I don't want to anything at the database level, because that would had a really heavy impact.

Reply:
Hm...